Description: Make Nafees Nastaleeq the preferred font for Urdu
Bug-Ubuntu: https://launchpad.net/bugs/1153188
Bug-Debian: https://bugs.debian.org/703456
 That bug is not really a forward of this patch. So far it's rather a
 discussion with Debian about adding the related .ttf file to begin with.
 The plan is to sort things out with upstream after the 14.04 release.
Author: Gunnar Hjalmarsson <gunnarhj@ubuntu.com>

--- fonts-nafees-1.2.orig/90-fonts-nafees.conf	1970-01-01 01:00:00.000000000 +0100
+++ fonts-nafees-1.2/90-fonts-nafees.conf	2014-03-26 17:04:35.042119545 +0100
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<!DOCTYPE fontconfig SYSTEM "fonts.dtd">
+<fontconfig>
+	<match target="pattern">
+		<test name="lang" compare="contains">
+			<string>ur</string>
+		</test>
+		<test qual="any" name="family">
+			<string>sans-serif</string>
+		</test>
+		<edit name="family" mode="prepend" binding="strong">
+			<string>Nafees Nastaleeq</string>
+		</edit>
+	</match>
+
+	<match target="pattern">
+		<test name="lang" compare="contains">
+			<string>ur</string>
+		</test>
+		<test qual="any" name="family">
+			<string>serif</string>
+		</test>
+		<edit name="family" mode="prepend" binding="strong">
+			<string>Nafees Nastaleeq</string>
+		</edit>
+	</match>
+</fontconfig>
